spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_variable_attributes_spec.rb

```diff
@@ -382,6 +405,48 @@ module Requestors # rubocop:disable Metrics/ModuleLength
             end
           end
         end
+
+        context 'NINO' do
+          it 'generates the national insurance number in means and merits' do
+            %i[global_means global_merits].each do |entity|
+              block = XmlExtractor.call(xml, entity, 'NI_NO')
+              expect(block).to have_text_response applicant.national_insurance_number
+              expect(block).not_to be_user_defined
+            end
+          end
+        end
+
+        context 'Proceeding limitation attrs' do
+          context 'delegated functions used' do
```

Comment:
can these 2 be refactored into one test as they both have the same expectations?